frontend/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="../src/images/quizzical_icon.svg" />
```

Comment:
I'm just wondering, see the change here? %PUBLIC_URL%, it looks like that's a shortcut for the public folder, so if you have static images and icons and stuff, I think maybe it's better to put them in that folder, maybe something to test?